README.rst

```diff
@@ -107,6 +107,37 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
 
 ----
 
+Compiling DALI from source (Docker) - recommended
+-------------------------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^
+
+.. |docker link| replace:: **Docker**
+.. _docker link: https://docs.docker.com/install/
+
+.. table::
+   :align: center
+
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | Linux x64                              |                                                                                             |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |docker link|_                         | Follow installation guide and manual at the link.                                           |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+
+Building Docker image
+^^^^^^^^^^^^^^^^^^^^^
+
+Change directory (``cd``) into Docker directory and run ``./build.sh``. If needed, set the following environment variables:
```

Comment:
What might be worthy to notice here is that if we provide a script (I mean, officially, i.e. put it in documentation), we should make sure it works (e.g. test it in CI)

Reply:
We use the same docker files as in CI, bash that calls them differs only slightly. manylinux3 could make a difference. I think we can add some L3 for that.